aztec/src/main/kotlin/org/wordpress/aztec/AztecParser.kt

```diff
@@ -591,15 +591,27 @@ class AztecParser {
         }
 
         if (nl == 0 && text.length > i && text[i] == '\n') {
-            consumeCursorIfThere(out, text, i)
+            consumeCursorIfInInput(out, text, i)
         }
     }
 
-    private fun consumeCursorIfThere(out: StringBuilder, text: CharSequence, position: Int) {
+    /**
+     * Append a cursor to the output string if input string has one at the specified position.
+     * Cursor is removed from the input if found at that position.
+     *
+     * The algorithm that uses this function goes like this: While traversing the input (spannable) string and producing
```

Comment:
Beginning with `The algorithm that uses this function goes like this:` is unnecessary.